content/en/continuous_integration/static_analysis/rules/_index.md

```diff
@@ -0,0 +1,337 @@
+  - link: "/continuous_integration/static_analysis/rules/python-django/use-convenience-imports"
+    tag: "use-convenience-imports"
+    text: "use convenience imports whenever possible"
+python_flask_data:
+  - link: "/continuous_integration/static_analysis/rules/python-flask/use-jsonify"
+    tag: "use-jsonify"
+    text: "use jsonify instead of json.dumps for JSON output"
+python_inclusive_data:
+  - link: "/continuous_integration/static_analysis/rules/python-inclusive/comments"
+    tag: "comments"
+    text: "check comments for wording issues"
+  - link: "/continuous_integration/static_analysis/rules/python-inclusive/function-definition"
+    tag: "function-definition"
+    text: "check function names for wording issues"
+  - link: "/continuous_integration/static_analysis/rules/python-inclusive/variable-name"
+    tag: "variable-name"
+    text: "check variable names for wording issues"
+further_reading:
+  - link: "/continuous_integration/static_analysis/"
+    tag: "Documentation"
+    text: "Learn about Datadog Static Analysis"
+---
+
+## Overview
+
+{{% site-region region="us,us3,us5,eu,ap1" %}}
+<div class="alert alert-warning">
+  Static Analysis is in private beta. Python is the only supported language. To request access, <a href="/help">contact Support</a>.
+</div>
+{{% /site-region %}}
+
+{{% site-region region="gov" %}}
+<div class="alert alert-danger">
+    Static Analysis is not available for the {{< region-param key="dd_site_name" >}} site.
+</div>
+{{% /site-region %}}
+
+Datadog Static Analysis provides out-of-the-box rules to help detect violations in your CI/CD pipelines in code reviews and identify bugs, security, and maintainability issues. For more information, see the [Static Analysis documentation][1].
+
+## Rules
+
+### Follow best practices for writing Python code
+
+**Ruleset ID:** `python-best-practices`
+
+Best practices for Python to write efficient and bug-free code.
+
+{{< partial name="continuous_integration/rule-list-python-best-practices.html" >}}
+
+<br>
+
+### Enforce Python code style
+
+**Ruleset ID:** `python-code-style`
+
+Rules to enforce Python code style
+
+{{< partial name="continuous_integration/rule-list-python-code-style.html" >}}
+
+<br>
+
+### Check Python program structure (e.g. nested loops, etc.)
+
+**Ruleset ID:** `python-design`
+
+Rules to check your Python program structure
+
+{{< partial name="continuous_integration/rule-list-python-design.html" >}}
+
+<br>
+
+### Rules specific for Django best practices and security
+
+**Ruleset ID:** `python-django`
+
+Rules made specifically for Django best practices and security
+
+{{< partial name="continuous_integration/rule-list-python-django.html" >}}
+
+<br>
+
+### Rules for Flask best practices and security
+
+**Ruleset ID:** `python-flask`
+
+Rules made specifically for Flask best practices and security
+
+{{< partial name="continuous_integration/rule-list-python-flask.html" >}}
+
+<br>
+
+### Check your Python code for wording issues
```

Comment:
```suggestion
### Check for wording issues
```

Reply:
I like to keep `Python` or a Python library (`Flask`, `Django`) in the heading for now because in the not too distant future, we'll start adding rulesets for different languages and this will help differentiate rulesets for now.